exercises/practice/pop-count/.npmrc

```diff
@@ -0,0 +1 @@
+audit=false
```

Comment:
This is also defined in the root and I suspect isn't needed

Reply:
Another auto-generated file.